New states June

featherus opened the pull request.

featherus committed:
Update Chinese State Borders

featherus committed:
Add Indonesian States

featherus committed:
Fix state catagories for Indonesia

Thinking-waffle committed:
Merge branch 'Beta-Edits' into NewStatesJune

Thinking-waffle committed:
Merge branch 'Beta-Edits' into NewStatesJune

Thinking-waffle committed:
Merge branch 'Beta-Edits' into NewStatesJune

marijn211 committed:
Merge branch 'Beta-Edits' into NewStatesJune

Thinking-waffle committed:
Merge branch 'Beta-Edits' into NewStatesJune

TheBrit7 commented:
It's difficult to accept a PR where 14 states are added/modified and aren't accounted for in terms of whatever focuses and formables, but due to the specific states I will assume this was requested.

RMD-Angel committed:
Adjustments for Indonesia

Added cities and railways to Indonesia and made it so that the Japanese focus tree now accommodates the new states. More will come after Jakarta is put into East/Central Java.

Made with permission from Featherus.

ilkober committed:
Merge branch 'Beta-Edits' into NewStatesJune

ilkober committed:
Fixing Daladier

ilkober committed:
Fixing formable and readding Kalimantan + 909/910 + Polish leader tweak

RMD-Angel committed:
State adjustments

Continued adjusting events, decisions, and national focuses to the new states except the Palembang decision for obvious reasons. Adjustments for Java will be done after Jakarta has either been included into a different state or East and Central Java are merged.

ilkober committed:
Fixing the fixes for 672 -> 672 + 905

ilkober committed:
906 secured

ilkober commented:
States in focuses were readjusted, naval bases accept ships and don't crash the game
In my opinion, the branch can be merged

ilkober committed:
little fix
